spotlessSelf.gradle

```diff
@@ -6,18 +6,19 @@ plugins {
 
 repositories { jcenter() }
 spotless {
+	def noInternalDepsClosure = {
+		if (it.contains('import org.gradle.internal.') &&
+		!it.contains('def noInternalDepsClosure')) {
```

Comment:
I think this condition does little if anything to help here.

With this condition in place, it means that if we format `spotlessSelf.gradle`, then it will never throw `AssertionError` even if it imports `org.gradle.internal.*`. I don't know if build files can even import Gradle-internal packages, but we might as well play it safe I reckon.